docs/bugs_and_glitches.md

```diff
@@ -2629,3 +2630,18 @@ If `IsInArray` returns `nc`, data at `bc` will be executed as code.
  	jr nz, .loop
  	ret
 ```
+
+
+### Options Menu fails to clear joypad state on initialization
+
+This bug (or feature!) results in all options being shifted left or right if the respective direction is pressed on the same frame the options menu is opened. The bug also exists in pokeyellow and pokegold.
```

Comment:
```suggestion
### Options menu fails to clear joypad state on initialization

This bug allows all the options to be updated at once if the left or right buttons are pressed on the same frame that the options menu is opened.
```

Reply:
Mine was simply a c&p of the wording in the pokeyellow docs. Yours wording is much better. Thanks.